src/api/app/policies/package_policy.rb

```diff
@@ -1,4 +1,18 @@
 class PackagePolicy < ApplicationPolicy
+  # FIXME: Using more than 2 arguments is considered a code smell.
+  def initialize(user, record, ignore_lock = false)
```

Comment:
I could have probably worked around this code smell (so having another argument, `ignore_lock`) by [using a context as explained in Pundit's README](https://github.com/varvet/pundit#additional-context), but this would have involved even more refactoring. I decided this was already good enough for this PR.